### PR TITLE
[DOCS]fix: all outgoing PACT links to the new internal ones

### PR DESCRIPTION
**Pull request by sstraatemans**

### CODE REVIEW

File: packages/libs/client/README.md [hunk 1]
```diff
@@ -535,7 +535,7 @@ We are available via Discord and Github issues:
 [27]:
   https://github.com/kadena-community/kadena.js/blob/main/packages/libs/client-examples/src/example-contract/simple-transfer.ts
 [28]:
-  https://pact-language.readthedocs.io/en/stable/pact-reference.html?highlight=signer#signature-capabilities
```
Comment (sstraatemans) [inline packages/libs/client/README.md:538]: @javadkh2 I got 1 change in your readme.
we are launching the new docs. so this will be the link.
I need an approval from you. if ok

Comment (javadkh2) [reply packages/libs/client/README.md:538]: Both links are broken!

Comment (sstraatemans) [reply packages/libs/client/README.md:538]: the link is broken for now. it will work when docs is going live!
is that ok for half a day?

I can make it go to: https://alpha-docs.kadena.io/pact/reference/concepts#signature-capabilitiesh289463870
for now and switch it tomorrow?

File: packages/apps/docs/src/pages/build/guides/building-a-voting-dapp.md [hunk 2]
```diff
@@ -244,8 +244,8 @@ Don't forget to add the snippet above in the `election` module body.
 :::info
 
 To learn more about guards and capabilities, please visit the
-[Guards, Capabilities and Events](/pact/reference/concepts#guards) section of
-the Pact official documentation.
+[Guards, Capabilities and Events](/pact/reference/concepts#guardsh2143724174)
+section of the Pact official documentation.
 
 :::
```
Comment (realdreamer) [inline packages/apps/docs/src/pages/build/guides/building-a-voting-dapp.md:251]: this file needs to be removed and have to update the new election-dapp sections here.

File: packages/apps/docs/src/pages/build/guides/pact-local-api-queries.md [hunk 3]
```diff
@@ -22,10 +21,10 @@ without necessarily having to spend tokens!
 
 Follow the instructions at the [Welcome to Pact](/pact/beginner) section to get
 started with Pact learning the basics. For a more straightforward and technical
-introduction, just [readthedocs](https://pact-language.readthedocs.io).
+introduction, just [readthedocs](/pact/reference).
```
Comment (realdreamer) [inline packages/apps/docs/src/pages/build/guides/pact-local-api-queries.md:24]:
```suggestion
introduction, just read the [docs](/pact/reference).
```

File: packages/libs/client/README.md (hunk 1 above)
Comment (javadkh2) [reply packages/libs/client/README.md:538]: That's fine if it will work eventually.

File: packages/apps/docs/src/pages/pact/beginner/hello-world.md [hunk 4]
```diff
@@ -106,11 +106,10 @@ our GitHub.
 ### Create a Keyset
 
 As you can see, the “Hello World” smart contract starts by defining and reading
```
Comment (realdreamer) [inline packages/apps/docs/src/pages/pact/beginner/hello-world.md:108]: Above "## Hello World Overview" section hyperlinks are not proper. (pact.kadena.io)[http://pact.kadena.io]. it should be [pact.kadena.io](http://pact.kadena.io)

it's not part of this PR. Since you're already making changes thought of adding it here

File: packages/apps/docs/src/pages/pact/beginner/hello-world.md [hunk 5]
```diff
@@ -164,8 +161,7 @@ module.
 ### Define a Function
 
 The next step is to create the contract functions. Functions are defined within
-Pact modules using the keyword
-(defun)[https://pact-language.readthedocs.io/en/latest/pact-reference.html#defun].
+Pact modules using the keyword (defun)[/pact/reference/syntax#defunh95462750].
```
Comment (realdreamer) [inline packages/apps/docs/src/pages/pact/beginner/hello-world.md:164]:
```suggestion
Pact modules using the keyword [defun](/pact/reference/syntax#defunh95462750).
```

**Review by javadkh2 — APPROVED** [review]

**Review by realdreamer — APPROVED** [review]
